README.md

```diff
@@ -1,4 +1,59 @@
 # SWIRL: Scientific Workflow Intermediate Representation Language
 
+SWIRL is a framework which allows the low-level compilation target for distributed workflow execution plans.
 
 
+## Install
+
+### PyPI
+Install SWIRL via [PyPi](https://pypi.org/project/swirl/)
+```bash
+pip install swirl
+```
+
+
+### Docker
+SWIRL Docker images are available on [Docker Hub](https://hub.docker.com/r/alphaunito/swirl).
+
+Example of `translate` option. Similar `compile` option with appropriate parameters.
+Next sections explain SWIRL options deeply.
+```bash
+mkdir -p SWIRL
+docker run                          \
+        --user $(id -u):$(id -g)    \
+        --volume $(pwd):/data       \
+        --workdir /data             \
+        alphaunito/swirl:0.0.1      \
+    swirl                           \
```

Comment:
You did not introduce neither translate nor compile at this point.
Substitute this with `swirl COMMAND [OPTIONS]`